javalang/parse.py

```diff
@@ -47,6 +47,12 @@ def parse_type_signature(sig):
 
     return parser.parse_class_or_interface_declaration()
 
+def parse_statement(stmt):
```

Comment:
So the intention is just to parse a complete statement like a block, and nothing more?